src/test/java/com/waverleysoftware/LoginToApplication.java

```diff
@@ -0,0 +1,32 @@
+package com.waverleysoftware;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class LoginToApplication extends BaseClass {
+    WebElement username = driver.findElement(By.id("email"));
```

Comment:
instance variables shouldn't be exposed to the entire package...
on the other hand, WebElements shouldn't be used on a test level... put them into PageObjects